Makefile

```diff
@@ -99,30 +99,46 @@ doc_R_pkg = \
 
 depends = .doc/$(1) .install/$(1) .check/$(1) .test/$(1)
 
+### Help
+
+help: ## Show this help message
```

Comment:
Please move down to appear below the `all:` target, so that `all` keeps being the default target (by virtue of being the first one in the file) and is what's run when someone types `make` with no target specified.

(I have a vague notion there might also be a way to explicitly mark a target as the default rather than relying on Makefile order, but I don't remember what that way is or how widely supported it is.)